src/sas/qtgui/Perspectives/Fitting/GPUOptions.py

```diff
@@ -76,7 +77,9 @@ def _addAllWidgets(self):
 
     def applyNonConfigValues(self):
         """Applies values that aren't stored in config. Only widgets that require this need to override this method."""
-        self.set_sas_open_cl()
+        if self._staged_open_cl:
```

Comment:
Maybe a comment that says why this has been done, so it doesn't accidentally get undone